taxcalc/current_law_policy.json

```diff
@@ -307,7 +307,7 @@
 		      "2015"],
         "cpi_inflated": true,
         "col_label": "",
-        "value":   [126,
```

Comment:
[Line 13 on this form.](http://taxhow.net/static/form/pdf/federal/1432440003/f8863--2013.pdf)